query-node/schema.graphql

```diff
@@ -12,32 +12,127 @@ type Block @entity {
   network: Network!
 }
 
+enum MembershipEntryMethod {
+  PAID
+  SCREENING
+  GENESIS
+}
+
 "Stored information about a registered user"
-type Member @entity {
+type Membership @entity {
   "MemberId: runtime identifier for a user"
   id: ID!
 
   "The unique handle chosen by member"
-  handle: String @unique @fulltext(query: "membersByHandle")
+  handle: String! @unique @fulltext(query: "membersByHandle")
 
   "A Url to member's Avatar image"
-  avatarUri: String
+  avatarUri: String!
 
   "Short text chosen by member to share information about themselves"
-  about: String
-
-  "Blocknumber when member was registered"
-  registeredAtBlock: Int!
+  about: String!
 
   "Member's controller account id"
-  controllerAccount: Bytes!
+  controllerAccount: String!
```

Comment:
Why the change from Bytes to String? (I assume Bytes would have represented the 32Byte raw account id and the string will be the encoded address?)

Reply:
I copied the membership from https://github.com/Joystream/joystream/pull/2105 where this change was present. I guess the point was to have it in some readable format for easy conversion to JSON, but I will ask Metin to double-check.

Reply:
The String is really used just for convenient work with the data in sandboxes, while building apps, etc.